src/taipy/core/notification/notifier.py

```diff
@@ -19,11 +19,38 @@
 
 def _publish_event(
     entity_type: EventEntityType,
```

Comment:
*note to reviewers*: the only two mandatory arguments are the `entity_type` and the `operation`. I propose to move them at first position, and be the only positional arguments for better readability. All the others can be specified using the named argument notation.